server/routes/events.ts

```diff
@@ -22,8 +22,15 @@ import {
   SpecialEvent,
   ProcessedSpecialEvent,
 } from "../types";
+//Logger
+import { logger } from "../loggerUtils/logger";
+import dotenv from "dotenv";
 
 const router = express.Router();
+dotenv.config();
+
+const TODAY =
+  process.env.NODE_ENV === "production" ? "TODAY()" : `"${process.env.TODAY}"`;
```

Comment:
TODAY should also default to TODAY() if "TODAY" environment variable wasn't set. The point of the env variable is to give the option (but not the requirement) to force the date.